cmd/struct-markdown/main.go

```diff
@@ -117,6 +117,8 @@ func main() {
 				fieldType = `duration string | ex: "1h5m2s"`
 			case "config.Trilean":
 				fieldType = `boolean`
+			case "hcl2template.NameValues":
+				fieldType = `repeatable "name" & "value" object`
```

Comment:
Here I'm trying to say in a super short sentence that this is a `[]{name string, value string }` but I'm not sure what's the best option for docs; do you have a suggestion ? May be I will try with `[]{name string, value string }`.